govuk-paas/paas-uaa-release
# Update to UAA 74.7

# What

Upgrades UAA to 74.7.0 to take advantage of the security fixes they've released.

# How to review

* Check that the commit history makes sense (i.e. that this is really a merge from upstream's uaa-release)
* Check that the src/uaa submodule really points at a sensible commit of paas-uaa (we rebased there, instead of merging, so the history should be nice and clean).

# Who can review

Not Rich or Mo.
